packages/contracts-bedrock/test/L1/OPContractsManager.t.sol

```diff
@@ -222,6 +222,10 @@ contract OPContractsManager_Upgrade_Harness is CommonTest {
             vm.skip(true);
         }
 
+        skipIfOpsRepoTest(
```

Comment:
`cd packages/contracts-bedrock && just clean && forge build && just gas-snapshot` should fix the failing ci.